flexmeasures/api/v3_0/tests/test_sensor_schedules_fresh_db.py

```diff
@@ -493,17 +493,18 @@ def test_multiple_contracts(
     print(consumption_schedule)
     print(soc_schedule)
 
-    # Check for absence of consumption breaches over 1 kW, i.e. any breach costs are avoided
-    assert all(v <= 0.001 for v in consumption_schedule)
+    # Check for consumption breaches over 0.25 kW, i.e. any breach costs are avoided as much as possible
```

Comment:
Not very clear what you mean w.r.t. any and all usage. We *expect* at least one breach, but also no large breaches?